tests/manage/configuration/migrations/test_migrations.py

```
@@ -76,9 +78,21 @@ def test_config_needs_migrating_incompatible_version():
         'CONFIG_VERSION': {'CURRENT': CURRENT_CONFIG_VERSION + 1, 'OLDEST_COMPATIBLE': CURRENT_CONFIG_VERSION + 1}
     }
 
-    with pytest.raises(ConfigurationVersionError, match=r'verdi config downgrade'):
+    with pytest.raises(ConfigurationVersionError, match=r'verdi config downgrade') as exception:
         check_and_migrate_config(config, filepath='/tmp/config.json')
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/config.json"

(S108)